client/src/components/mixins/date-mixin.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component
+export default class DateMixin extends Vue {
+  readonly MS_IN_A_DAY = (1000 * 60 * 60 * 24)
+
+  /**
+   * The number of days that 'date' is from today.
+   * @returns -1 for yesterday
+   * @returns 0 for today
+   * @returns +1 for tomorrow
+   * @returns NaN in case of error
+   */
+  daysFromToday (date: string): number {
```

Comment:
This is the same method used in Filings UI and Create UI.

Reply:
I tested this locally but when we have more time (!) then we should create a unit test for this.

(It works fine since both today and the date passed in are fully qualified, eg "Wed Dec 02 2020 08:00:00 GMT-0800" becomes Dec 2 00:00h local.)

Reply:
Very cool